profiles/common/modules/features/nexteuropa_dgt_connector/tmgmt_poetry/plugins/tmgmt_poetry.ui.inc

```diff
@@ -303,8 +303,8 @@ class TMGMTPoetryTranslatorUIController extends TMGMTDefaultTranslatorUIControll
       '#default_value' => (!empty($job->settings['contacts']['responsable']) ? $job->settings['contacts']['responsable'] : $contacts['responsable']),
     );
 
-    // Load translator settings : Organization.
-    $organization = $translator->getSetting('organization');
+    // Load translator settings from submitted job: Organization.
+    $organization = ($job->settings['organization']) ? $job->settings['organization'] : $translator->getSetting('organization');
```

Comment:
Here I would suggest we check whether or not the array key "organization" exists as from tmgmt the setting is an array and the key is not set by default, so nothing guarantee that it always exists.

Reply:
Hi @aritomelo I updated  the code in order to avoid the 'undefined index" on the array.
Please check, ammend and validatre. 
K.R.